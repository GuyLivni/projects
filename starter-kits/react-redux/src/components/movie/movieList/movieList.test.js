/* eslint-disable no-undef */
import renderer from "react-test-renderer";
import React from "react";
import MovieList from "./";

describe("Movie List", () => {
  it("MovieList ui renders correctly", () => {
    const tree = renderer.create(<MovieList movies={[]} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
