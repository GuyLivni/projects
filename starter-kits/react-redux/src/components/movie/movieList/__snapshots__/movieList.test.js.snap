// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Movie List MovieList ui renders correctly 1`] = `
<div
  className="ui stackable three cards"
/>
`;
